src/sentry/models/group.py

```diff
@@ -419,22 +391,6 @@ def get_latest_event_for_environments(self, environments=()):
             issue_id=self.id,
             project_id=self.project_id)
 
-    def get_oldest_event(self):
```

Comment:
Could plugins in sentry-plugins or other plugin repos be using this?

Reply:
Good point. I checked the whole Sentry org on github and there doesn't seem to be any other usages of this